examples/networkpolicy/kubernetes-calico.json

```diff
@@ -10,19 +10,13 @@
     "masterProfile": {
       "count": 1,
       "dnsPrefix": "",
-      "vmSize": "Standard_D2_v2"
+      "vmSize": "Standard_DS2_v2"
```

Comment:
Any reason for the change to the VM SKU in the example api model?

Reply:
Yes, this uses the faster premium storage based VMs.